tests/end-to-end/helpers/util.js

```diff
@@ -478,6 +477,7 @@ async function createBasicField(t, field) {
         await t.click(editFieldModal.getToggle('Others option'))
       }
       break
+    case 'checkbox':
```

Comment:
Need to add in 'others option' here for checkbox! Otherwise, the e2e test won't be able to find the 'others' checkbox later on:

```
    case 'checkbox':
      await t.selectText(editFieldModal.optionTextArea).pressKey('delete')
      // The wait(500) is necessary because of the debounce time in reloadDropdownField
      await t
        .typeText(editFieldModal.optionTextArea, field.fieldOptions.join('\n'))
        .wait(500)
      if (field.othersRadioButton) {
        await t.click(editFieldModal.getToggle('Others option'))
      }
      break
    case 'dropdown':
      await t.selectText(editFieldModal.optionTextArea).pressKey('delete')
      // The wait(500) is necessary because of the debounce time in reloadDropdownField
      await t
        .typeText(editFieldModal.optionTextArea, field.fieldOptions.join('\n'))
        .wait(500)
      break
```

Reply:
Thx